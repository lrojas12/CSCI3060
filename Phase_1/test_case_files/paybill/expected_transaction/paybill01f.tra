10                                     A 
03 Tarzan               00001 00100.00 TV
00                                     A 