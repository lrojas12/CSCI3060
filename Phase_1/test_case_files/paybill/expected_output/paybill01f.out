Welcome to Watermelon Banking System
Please login to begin or type in “help” for more information.
Mode in which you wish to login as:
You are currently logged in as an administrator.
Enter origin account holder’s name:
Enter company:
Enter amount to pay:
You have successfully paid a bill of 00100.00 from account 00001 to TV. 
New balance: $00900.00
ERROR: You may not pay any bills using this account.
You have successfully logged out of the administrator account 